modin/pandas/test/test_groupby.py

```diff
@@ -1500,6 +1500,24 @@ def test_dict_agg_rename_mi_columns(
     df_equals(md_res, pd_res)
 
 
+def test_agg_4604():
+    data = {"col1": [1, 2], "col2": [3, 4]}
+    md_df, pd_df = pd.DataFrame(data), pandas.DataFrame(data)
```

Comment:
```suggestion
    modin_df, pandas_df = pd.DataFrame(data), pandas.DataFrame(data)
```
I prefer using concrete prefixes for dataframes. We recommend importing Modin as `import modin.pandas as pd`, but not as `md`, as there might be such a confusion.